cmake/jana_plugin.cmake

```diff
@@ -223,37 +219,36 @@ macro(plugin_add_event_model _name)
 
     if(NOT EDM4EIC_FOUND)
         find_package(EDM4EIC REQUIRED)
-        set(EDM4EIC_INCLUDE_DIR ${EDM4EIC_DIR}/../../include)
     endif()
 
     # Add include directories
-    # (same as target_include_directories but for both plugin and library)
     # ${podio_BINARY_DIR} is an include path to datamodel_glue.h
-    plugin_include_directories(${PLUGIN_NAME} SYSTEM PUBLIC ${podio_INCLUDE_DIR} ${EDM4EIC_INCLUDE_DIR} ${EDM4HEP_INCLUDE_DIR} ${podio_BINARY_DIR})
+    plugin_include_directories(${PLUGIN_NAME} PUBLIC ${podio_BINARY_DIR})
```

Comment:
Actually, this doesn't do anything. `podio_BINARY_DIR` isn't a thing.

Reply:
It's a thing. CMake defines <projectname>_BINARY_DIR. This allows to reference podio plugin independently of what is the top level project.

Reply:
Right. I meant that we're not writing anything to `${podio_BINARY_DIR}` anymore in `datamodel_glue`.

Reply:
But we should. Let me redo this.